shelley/design-spec/delegation_design_spec.tex

```diff
@@ -189,6 +189,9 @@
   affects the implementation.}
 \change{2020-07-23}{PK}{FM (IOHK)}{Change: undistributed rewards go to the
   reserves, not to the treasury.}
+\change{2020-10-08}{PK, DC}{FM (IOHK)}{Change: include member stake in the non-myopic
```

Comment:
I can make this `J. Corduan` or `JC` or `PK, DC, JC` or whatever is best :smile:

Reply:
Seems like you should at least credit yourself, unless you don't want the fan mail :-)

Reply:
ok, I'm on the list now :)